examples/server/server.c

```diff
@@ -354,37 +354,148 @@ static int NonBlockingSSL_Accept(SSL* ssl)
+    }
+
+    while (throughput > xfer_bytes) {
+        select_ret = tcp_select(clientfd, 1); /* Timeout=1 second */
+        if (select_ret == TEST_RECV_READY) {
+            len = min(block, (int)(throughput - xfer_bytes));
+            rx_pos = 0;
+
+            while (rx_pos < len) {
+                start = current_time(1);
+                ret = SSL_read(ssl, &rx_buffer[rx_pos], len - rx_pos);
+                rx_time += current_time(0) - start;
+                if (ret < 0) {
+                    err = SSL_get_error(ssl, 0);
+                #ifdef WOLFSSL_ASYNC_CRYPT
+                    if (err == WC_PENDING_E) {
+                        ret = wolfSSL_AsyncPoll(ssl, WOLF_POLL_FLAG_CHECK_HW);
+                        if (ret < 0) break;
+                    }
+                    else
+                #endif
+                    if (err != WOLFSSL_ERROR_WANT_READ &&
+                                             err != WOLFSSL_ERROR_ZERO_RETURN &&
+                                             err != APP_DATA_READY) {
+                        err_sys_ex(runWithErrors, "ServerBenchmarkThroughput SSL_read failed");
+                        break;
+                    }
+                    if (err == WOLFSSL_ERROR_ZERO_RETURN) {
+                        free(rx_buffer);
+                        free(tx_buffer);
+                        return WOLFSSL_ERROR_ZERO_RETURN;
+                    }
+                }
+                else {
+                    rx_pos += ret;
+                }
+            }
+            xfer_bytes += ret;
+        }
+    }
+
+    xfer_bytes = 0;
+    while (throughput > xfer_bytes) {
+        len = min(block, (int)(throughput - xfer_bytes));
+        /* Write data */
+        do {
+            err = 0; /* reset error */
+            start = current_time(1);
+            ret = SSL_write(ssl, tx_buffer, len);
+            tx_time += current_time(0) - start;
+            if (ret <= 0) {
+                err = SSL_get_error(ssl, 0);
+            #ifdef WOLFSSL_ASYNC_CRYPT
+                if (err == WC_PENDING_E) {
+                    ret = wolfSSL_AsyncPoll(ssl, WOLF_POLL_FLAG_CHECK_HW);
+                    if (ret < 0) break;
+                }
+            #endif
+            }
+        } while (err == WC_PENDING_E);
+        if (ret != len) {
+            err_sys_ex(runWithErrors, "ServerBenchmarkThroughput SSL_write failed");
+        }
+        
+        xfer_bytes += ret;
+    }
+
+    free(rx_buffer);
+    free(tx_buffer);
+
+    printf(
+    #if !defined(__MINGW32__)
+        "wolfSSL Server Benchmark %zu bytes\n"
+    #else
+        "wolfSSL Server Benchmark %d bytes\n"
+    #endif
+        "\tRX      %8.3f ms (%8.3f MBps)\n"
+        "\tTX      %8.3f ms (%8.3f MBps)\n",
+#if !defined(__MINGW32__)
+        throughput,
+#else
+        (int)throughput,
+#endif
+        rx_time * 1000, throughput / rx_time / 1024 / 1024,
+        tx_time * 1000, throughput / tx_time / 1024 / 1024
```

Comment:
Looks like you flipped the order of these parameters - I think that explains the change to matching numbers.

Reply:
Wow yeah that could very well be it lol. There goes my explanation. Let me confirm that. If you're right, I can probably just ditch this PR altogether.

Reply:
Confirmed.